File: killrvideo-service-comments/src/main/java/com/killrvideo/service/comment/dao/CommentDaoConfig.java
package com.killrvideo.service.comment.dao;

import com.datastax.oss.driver.api.core.CqlSession;
import org.springframework.context.annotation.Bean;
import org.springframework.context.annotation.Configuration;

@Configuration
public class CommentDaoConfig {
    @Bean
    public CommentMapper commentMapper(CqlSession session) {
        return CommentMapper.build(session).build();
    }
}

File: killrvideo-service-comments/src/main/kotlin/com/killrvideo/service/comment/dao/CommentDaoConfig.kt
package com.killrvideo.service.comment.dao

import com.datastax.oss.driver.api.core.CqlSession
import org.springframework.context.annotation.Bean
import org.springframework.context.annotation.Configuration

@Configuration
open class CommentDaoConfig {
    @Bean
    open fun commentMapper(session: CqlSession?): CommentMapper? {
        return CommentMapperBuilder(session).build()
    }
}
